### DISCUSSION TITLE
🔗 Too Much Space Below Testimonial Carousel (Duplicate of #22173)

### DISCUSSION THREAD

**Discussion by bt46n2** [elementor/elementor#24040] [Feature Request]
### Prerequisites

- [ ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I contacted customer support with this issue and they had no workaround and suggested I come here. When a customer submits a testimonial or review, I have no control over how long it is, but when adding them to the Testimonial Carousel, one or two will display looking nice, the others (due to different word counts) can have what seems like near infinite blank space below them, especially when on mobile. I have my call to action to covert prospective clients just below the reviews, but from a new prospective client's view point, they'd have to scroll and scroll and scroll through seemingly endless blank space before reaching the Call to action / subscribe button below.

I was hoping customer support could recommend some CSS to remedy this issue, but they say there is none. While customer testimonials are invaluable, the extreme amount of blank space below (especially on mobile) makes it almost useless in getting them to the call to action. It's quite frustrating. I've attached two screen shots. One looks perfect and leads right to the call to action below, the other you have to scroll and scroll and scroll through tons of blank space to reach the call to action.  
![testimonial caro  1](https://github.com/elementor/elementor/assets/114094946/68c068a9-4e70-40a4-a70e-d8efd497b68a)

![testimonial 2](https://github.com/elementor/elementor/assets/114094946/6270622c-3b73-42f8-abec-fd692a28e47d)

### Describe the solution you'd like

I'd like to see the blank space resolved either by CSS (ideally so it can be done quickly) and/or in the furutre.

### Describe alternatives you've considered

I've had to break up a single review in to two reviews to try to reduce some of the blank space

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #22173

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
